extensions/wikia/EnableDiscussions/EnableDiscussionsHooks.class.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+
+class EnableDiscussionsHooks {
+
+	const DISCUSSIONS_URL = '/d/f';
+	// Forum namespaces should be available
+	// because both Forum.setup and ForumDisabled.setup define them
+	private static $forumNamespaces = [
+		NS_WIKIA_FORUM_BOARD,
+		NS_WIKIA_FORUM_TOPIC_BOARD,
+		NS_WIKIA_FORUM_BOARD_THREAD,
+	];
+
+
+	public static function onBeforeInitialize(
+		Title $title = null, $unused, OutputPage $output = null, $user, $request, $mediaWiki
+	) {
+		if ( !self::shouldRedirectToDiscussions() || $title == null ) {
+			return true;
+		}
+		$forumTitle = self::getForumTitle( $title );
+		if ( empty( $forumTitle ) ) {
+			return true;
+		}
+
+		$output->redirect( self::getDiscussionsUrl( $forumTitle->getNamespace(),
+			$forumTitle->getArticleID() ) );
```

Comment:
Yeah, that's how this code style reformats it. Extracted the url to a separate variable.